### ISSUE TITLE
Player's title and buttons not visible in blurred song cover background and light mode depending on album cover

### ISSUE THREAD

**Issue by Crowerade** [fast4x/RiMusic#5096]
### Version of RiMusic

0.6.65.1

### Android version

14

### I've done these:

- [X] Uninstall old build before installing a new one
- [X] Checked opening issues about my similar problem

### Tell us how you trigger this bug

1. Go to settings
2. Got to "UI"
3. Select "Light" for the theme mode
4. Go to "Player Appearance"
5. Select "Blurred song cover" for the background colors
6. Go back from settings
7. Select a song with a dark album cover
8. Open the player
9. The title, artist name and control buttons are almost invisible

### Submit a video

![Screenshot_20241228-101838_RiMusic](https://github.com/user-attachments/assets/54a85bb9-074d-4b75-ae83-714a91c96c7e)
![Screenshot_20241228-102040_RiMusic](https://github.com/user-attachments/assets/256c9696-5d75-41e5-a2d1-6542fd239e06)

### What was your expectation?

To see the title, artist name and control buttons clearly enough

**Comment by aneesh1122** [comment]
Enable text outline from settings

**Comment by Crowerade** [comment]
I did but it's not as pretty, I think it would be possible to have the title, etc, in a colour adapted from the one extracted from the cover, that would make it more visible.
I understand this might not be a priority, if needed, we could change the tag from bug to enhancement?

**Comment by aneesh1122** [comment]
> I did but it's not as pretty

Use dark theme then

> I think it would be possible to have the title, etc, in a colour adapted from the one extracted from the cover, that would make it more visible.

Album cover may have many different colors on it. How can you find one specific color which will be visible over that particular area? If there was one background color then I would have used the opposite color of it.

You have many options to choose from.

1) don't use light theme
2) don't use blurred album cover as background
3) use text outline

**Comment by Crowerade** [comment]
I agree that it might be fixed if I tweak some things but I think it's shame that some components do not work well together although they have a lot of potential.
Otherwise it would mean that I can only use blurred cover as background in dark theme (since text outline does not apply to control buttons and action bar gradient does not seem to do anything), but not everyone likes it

**Comment by aneesh1122** [comment]
> action bar gradient does not seem to do anything

Same settings as yours but I also enabled the action bar gradient and it's working fine.

![Screenshot_2024-12-29-19-38-07-646_it.fast4x.rimusic.mgri_debug.jpg](https://github.com/user-attachments/assets/bc1a4534-d81a-446f-91e7-2002474098ce)

![Screenshot_2024-12-29-19-39-32-782_it.fast4x.rimusic.mgri_debug.jpg](https://github.com/user-attachments/assets/a304c813-8f0c-44fe-bdcc-806d8d4b948b)


You have to enable this

![Screenshot_2024-12-29-19-39-52-986_it.fast4x.rimusic.mgri_debug-edit.jpg](https://github.com/user-attachments/assets/13d103dc-0db8-4216-816b-7b33e1eca2a3)

**Comment by Crowerade** [comment]
I already did as said in my previous comment and it does not change anything on my screen
